### ISSUE TITLE
Loosen (or remove) peerDependency on graphql

### ISSUE THREAD

**Issue by alanhussey** [gajus/babel-plugin-graphql-tag#8]
I'm trying to integrate `babel-plugin-graphql-tag` into my project, but the peerDependency version of `^0.10.1` conflicts with my installed version of `0.11.7`, and prevents me from upgrading to a `0.12.x` release. 

One resolution to this would be to broaden the accepted peerDependency range, [like `graphql-tag` does](https://github.com/apollographql/graphql-tag/blob/384a9b222a67bdcc966ed23a2c43bbbcd26db10d/package.json#L33). This would allow apps (like mine) that depend on `babel-plugin-graphql-tag` to upgrade.

However, from a search through this repo it seems like `graphql` is never explicitly imported, which makes me suspect that it is not needed as a peerDependency. Since this project only directly imports `graphql-tag`, and `graphql-tag` has its own peerDependency on `graphql`, it should be sufficient to let `graphql-tag` declare its own peerDependency on `graphql`.

I'll post a PR to remove the peerDependency shortly. If you'd prefer to keep the peerDependency and simply broaden it, I'm happy to submit a PR for that instead. 

Relates to https://github.com/gajus/babel-plugin-graphql-tag/issues/7

**Comment by alanhussey** [comment]
Yup, https://github.com/gajus/babel-plugin-graphql-tag/issues/9

**Comment by jaydenseric** [comment]
It looks like this has been released in [v1.6.0](https://github.com/gajus/babel-plugin-graphql-tag/releases/tag/v1.6.0). This issue can be closed?

**Comment by mathix420** [comment]
`graphql` needs to be in `peerDependancies` in order to avoid conflicts in versions.

Removing `graphql` and `graphql-tag` from `devDependencies` and adding this fix below solved my issue.
```json
  "peerDependencies": {
    "@babel/core": "^7.0.0",
    "graphql-tag": "*",
    "graphql": "*"
  },
```

```
Ensure that there is only one instance of "graphql" in the node_modules directory. If different versions of "graphql" are the dependencies of other relied on modules, use "resolutions" to ensure only one version is installed. https://yarnpkg.com/en/docs/selective-version-resolutions Duplicate "graphql" modules cannot be used at the same time since different versions may have different capabilities and behavior. The data from one version used in the function from another could produce confusing and spurious results.
```
